src/include/security/dialogs.rb

```diff
@@ -973,5 +965,18 @@ def LoginDialog
 
       deep_copy(ret)
     end
+
+    def boot_vbox_widgets
+      VBox(
+        VSpacing(1),
+        Arch.s390 ? Empty() : settings2widget("CONSOLE_SHUTDOWN"),
```

Comment:
I'm afraid that this is wrong, we want to offer this combobox, but we do not want to have "reboot" there. Just "Ignore" and "Halt".